lib/resources/example_json/r4_examples_medication_request.rb

```diff
@@ -164,6 +341,3830 @@ module Resources
+                 Refills: 0.',
+                'timing': {
+                  'repeat': {
+                    'boundsPeriod': {
+                      'start': '2020-07-14T09:45:00.000-05:00'
+                    }
+                  },
+                  'code': {
+                    'coding': [
+                      {
+                        'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003',
+                        'code': '696547',
+                        'display': 'every 4 hr'
+                      },
+                      {
+                        'system': 'http://terminology.hl7.org/CodeSystem/v3-GTSAbbreviation',
+                        'code': 'Q4H',
+                        'display': 'Q4H'
+                      },
+                      {
+                        'system': 'http://snomed.info/sct',
+                        'code': '225756002',
+                        'display': 'Every four hours (qualifier value)'
+                      }
+                    ],
+                    'text': 'every 4 hr'
+                  }
+                },
+                'asNeededCodeableConcept': {
+                  'text': 'as needed for pain'
+                },
+                'route': {
+                  'coding': [
+                    {
+                      'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4001',
+                      'code': '318185',
+                      'display': 'Oral',
+                      'userSelected': true
+                    },
+                    {
+                      'system': 'http://snomed.info/sct',
+                      'code': '26643006',
+                      'display': 'Oral route (qualifier value)',
+                      'userSelected': false
+                    }
+                  ],
+                  'text': 'Oral'
+                },
+                'doseAndRate': [
+                  {
+                    'doseQuantity': {
+                      'value': 2,
+                      'unit': 'cap',
+                      'system': 'http://unitsofmeasure.org',
+                      'code': '{Capsule}'
+                    }
+                  }
+                ]
+              }
+            ],
+            'dispenseRequest': {
+              'validityPeriod': {
+                'start': '2020-07-14T09:45:15.000-05:00'
+              },
+              'numberOfRepeatsAllowed': 0,
+              'quantity': {
+                'value': 90,
+                'unit': 'cap',
+                'system': 'http://unitsofmeasure.org',
+                'code': '{Capsule}'
+              }
+            },
+            'substitution': {
+              'allowedBoolean': true
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314010037',
```

Comment:
These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Reply:
comment addressed